### ISSUE TITLE
Backintime not starting on POP OS 22.04

### ISSUE THREAD

**Issue by julienlarena** [bit-team/backintime#1250]
After update to POP OS 22.04, Backintime does not work anymore. Output after entering backintime on CL is:
```raceback (most recent call last):
  File "/usr/share/backintime/common/backintime.py", line 27, in <module>
    import config
  File "/usr/share/backintime/common/config.py", line 32, in <module>
    import tools
  File "/usr/share/backintime/common/tools.py", line 1802, in <module>
    class OrderedSet(collections.MutableSet):
AttributeError: module 'collections' has no attribute 'MutableSet'```

**Comment by emtiu** [comment]
Thank you for reporting! This is a known issue, and should be fixed with backintime 1.3.2 – if you have a way of nudging POP OS to include this latest version in their repositores, that would help!

**Comment by julienlarena** [comment]
Thanks. I reported that to them too and mentioned the fix in 1.3.2.

**Comment by emtiu** [comment]
Okay, thanks! I'm closing this issue for now, but you are welcome to reopen if the situation changes.
